projects/plugins/boost/compatibility/page-optimize.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
Does this need any code to prevent PageOptimize from running if our feature is on? Is it dangerous to run them side-by-side?

Reply:
Dangerous - no. But if you have Page Optimize JS concat enabled and disable Boost's, JS will still be concatenated.

I've made an update [here](https://github.com/Automattic/jetpack/pull/30005/commits/3e811f632fc719deb0c9d476db1383411ab9bf47), though this means, Page Optimize functionality will only be disabled if JS or CSS concatenation is enabled.

Should we always disable Page Optimize's functionality if Boost is active? That makes more sense, considering people shouldn't be running say Boost Critical CSS and JS/CSS minification from Page Optimize. 

This also leads me to another thing (can't recall if we discussed that) - if there's a website running Page Optimize and they are using JS concatenation and they add Boost - should we enable the concatenation in Boost and disable the one in Page Optimize?

Reply:
> Dangerous - no. But if you have Page Optimize JS concat enabled and disable Boost's, JS will still be concatenated.

Ok. My only concern is whether it's dangerous when both are running. :) I don't mind if you have two plugins doing the same thing, but only turn one off - that the other keeps doing it. That's fine.

> This also leads me to another thing (can't recall if we discussed that) - if there's a website running Page Optimize and they are using JS concatenation and they add Boost - should we enable the concatenation in Boost and disable the one in Page Optimize?

I think we should turn the equivalent features on in Boost (won't that be handled by the compatibility stuff you've already written)? But I don't think we should change the settings in Page Optimize. So long as it's not dangerous to have both there. :)

Reply:
Thanks. The compatibility has been [improved](https://github.com/Automattic/jetpack/compare/69f2c6c7a3c66e2f67b449e451e244687506fb9c...6e35ffe05a9fe647d4cad31a430a541e01f41540) with a migration of all options as soon as Boost is activated and the Page Optimize plugin is active on the website.